app/src/main/java/com/infomaniak/drive/views/FileInfoActionsView.kt

```diff
@@ -598,8 +614,8 @@ class FileInfoActionsView @JvmOverloads constructor(
         }
 
         @CallSuper
-        fun deleteFileClicked() {
-            Utils.confirmFileDeletion(context, fileName = currentFile.name) { dialog ->
+        fun deleteFileClicked() = currentFile?.apply {
+            Utils.confirmFileDeletion(context, fileName = name) { dialog ->
```

Comment:
Don't use an `apply` if you're only applying once, use a `let` if you're only checking for it's non nullability